packages/extension-chakra-storefront/src/pages/product-detail/index.jsx

```diff
@@ -272,22 +263,20 @@ const ProductDetail = () => {
         } else {
             toast({
                 title: formatMessage(TOAST_MESSAGE_ALREADY_IN_WISHLIST),
-                status: 'info',
-                action: (
-                    <Button variant="link" onClick={() => navigate('/account/wishlist')}>
-                        {formatMessage(TOAST_ACTION_VIEW_WISHLIST)}
-                    </Button>
-                )
+                type: 'info',
+                action: {
+                    label: formatMessage(TOAST_ACTION_VIEW_WISHLIST),
+                    onClick: () => navigate('/account/wishlist')
```

Comment:
This may change the look from a link button to a default button of chakra v3? Can we keep the button element here? ![image](https://github.com/user-attachments/assets/112b62cd-92c4-422e-b2c2-a710463b2202)

Can we do this since we are rendering label as children in the Toaster Component?
```
action: {
    label: <Button variant="link" onClick={() => navigate('/account/wishlist')}>
        {formatMessage(TOAST_ACTION_VIEW_WISHLIST)}</Button>,
    onClick: () => navigate('/account/wishlist')
    }
```

Reply:
Nvm, It was not corret. I was trying to ask if there is a way to customise the default styling for ToastTrigger.

In pwa-kit app, we are rendering the trigger as a link button. At the moment, it is a button style. 
![image](https://github.com/user-attachments/assets/4f746139-fd4e-448d-8961-356a803c9abc)